src/main.c

```diff
@@ -13,6 +15,23 @@
 
 int main(int argc, char **argv)
 {
+    char input[50];
+    char loopEndstr[] = "quit";
+    int  endLoop      = false;
```

Comment:
```c
const char *quit = "quit";
int end_loop = false;
```
Stick to lower case snake_case.

Defining a constant string you would use `const char *`, variable arrays like you used here are less efficient as it has to calculate the arrays length. Whereas a `char *` points to a memory address where this constant string is.

In C remember hardcoded strings like this are null terminated automatically.